src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -371,6 +380,7 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
     };
     cy.scratch(CytoscapeGlobalScratchNamespace, globalScratchData);
 
+    console.log('Start Batch - CyGraph');
```

Comment:
these should be console.debug

Reply:
yes, I had a comment way above saying that console logging needed to be removed before the DNM label is removed.   I think it's probably fine to remove it now, so I will...